Core/views.py

```diff
@@ -87,20 +88,33 @@ class RegistrationView(FormView):
     success_url = reverse_lazy('home')
 
     form_class = RegistrationForm
-
+    
     event = Event.objects.filter(active=True).first()
+    # event=''
 
     def post(self, request, *args, **kwargs):
         form = RegistrationForm(request.POST)
+        print(((dict(request.POST))['student_number'])[0])
         if form.is_valid():
-            form.save()
-            messages.add_message(request, messages.SUCCESS,
+            try:
+                Registration.objects.get(student_number=((dict(request.POST))['student_number'])[0], event=self.event)
+                print("1")
```

Comment:
Remove this kind of print statements. We don't want to have this in our server logs.